README.md

```diff
@@ -88,6 +83,9 @@ customize it to include the relevant Test Kits and content for your platform.
 * This will allow you to rebuild using Jekyll locally, and serve a copy of the static files
 * You can run `bundle exec rake web:serve` to generate the files and view them at `http://localhost:4000`
 * Alternatively, you can run `docker compose run inferno_web bundle exec rake web:generate` to only generate the files
```

Comment:
This sections a bit confusing because you go back and forth between doing it on Docker and doing it locally. Maybe it should all just be on Docker since this seems very production oriented?

I tried all commands locally and via Docker and they all worked, so this section is still correct.

Reply:
Hmm, i'm not sure why i did the docker version in here.  I removed that.